app/do.js

```diff
@@ -5,18 +5,20 @@ ipc.on('show', function (event, message) {
   doItGetTheTimeDoItNow();
 })
 
+var minutesOfDay = function(m) {
+  return m.minutes() + m.hours() * 60;
+}
+
 function doItGetTheTimeDoItNow() {
-  var now = moment().tz('Asia/Taipei');
+  var now = minutesOfDay(moment().tz('Asia/Taipei'));
```

Comment:
Oh so this `now` is used later to display the time (with a `now.format()`). I think you need a different variable either here or later.